source/agora/test/Base.d

```diff
@@ -907,7 +907,103 @@ public class TestAPIManager
             ex.line = line;
             throw ex;
         }
+        retryFor(nodes[0].client.getBlocksFrom(0, 1).length > 0, 2.seconds);
```

Comment:
What's the purpose of this call? There is always at least a genesis block loaded in the node. You don't need discovery in order to determine this. It should be true before discovery.

Reply:
My mistake, I have removed it.